test/analyzer_test.js

```diff
@@ -145,4 +145,46 @@ suite('Analyzer', () => {
     });
   });
 
+  test('the analyzer stream will emit an error when one occurs during analysis', (done) => {
+    const root = path.resolve('test/static/project-analysis-error');
+    const analyzer = new StreamAnalyzer(
+      root,
+      path.resolve(root, 'index.html'),
+      null,
+      [],
+      [path.join(root, '**')]
+    );
+
+    mergeStream(
```

Comment:
Consider faking stderr with a memory stream and adding a simple assert.matches to make sure that the error output includes the error you expect.